#! /bin/sh -e
## 20_Makefile_lib64.dpatch by Craig Small <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removed entry about /lib64 as it breaks the sparc

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@

[mkl: ported to procps-3.2.8]
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: procps-3.2.8/Makefile
===================================================================
--- procps-3.2.8.orig/Makefile
+++ procps-3.2.8/Makefile
@@ -30,7 +30,7 @@ ln_sf    := ln -sf
 install  := install -D --owner 0 --group 0
 
 # Lame x86-64 /lib64 and /usr/lib64 abomination:
-lib64    := lib$(shell [ -d /lib64 ] && echo 64)
+lib64    := lib
 
 usr/bin                  := $(DESTDIR)/usr/bin/
 bin                      := $(DESTDIR)/bin/
@@ -127,7 +127,7 @@ check_gcc = $(shell if $(CC) $(ALL_CPPFL
 # produce separate executables for ppc and ppc64, s390 and s390x,
 # i386 and x86-64, mips and mips64, sparc and sparc64, and so on.
 # Failure to do so will cause data corruption.
-m64 := $(call check_gcc,-m64,$(call check_gcc,-mabi=64,))
+#m64 := $(call check_gcc,-m64,$(call check_gcc,-mabi=64,))
 ALL_CFLAGS += $(m64)
 
 ALL_CFLAGS += $(call check_gcc,-Wdeclaration-after-statement,)
